applications/FemToDemApplication/python_scripts/MainCouplingFemDem.py

```diff
@@ -24,8 +24,18 @@ class MainCoupledFemDem_Solution:
     def __init__(self, Model, path = ""):
         self.model = Model
         # Initialize solutions
+
+        if path == "":
+            DEM_json_file = open("MaterialsDEM.json",'r')
+            DEMProjectParametersFile = open("ProjectParametersDEM.json",'r')
+        else:
+            DEM_json_file = open(path + "MaterialsDEM.json")
+            DEMProjectParametersFile = open(path + "ProjectParametersDEM.json",'r')
```

Comment:
Better use `os.path.join(path, "MaterialsDEM.json")` to avoid problems with '\\' or '/'